File: Cap 04/4.6 - Operadores Logicos/4.6.1 - Operador Logico de Conjuncao/andtest.java
/*
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: 4 - Topico 4.6.1
 * 
 * Programa ..: OPERACAO DE CONJUNCAO ".E. / AND"
 */

import java.util.Scanner;

public class Main {
	
  static int NUMERO;
  
  public static void main(String[] args) {
	  
    Scanner entrada = new Scanner(System.in);
    
    System.out.println("TESTE LOGICO OPERADOR: AND");
    System.out.println();
    
    System.out.print("Entre um numero inteiro: ");
    NUMERO = entrada.nextInt();
    
    if (NUMERO >= 20 && NUMERO <= 90) {
      System.out.println("O numero esta na faixa de 20 a 90");
    } else {
      System.out.println("O numero esta fora da faixa de 20 a 90");
    }
    
    entrada.close();
    
  }
}

File: Cap 04/4.6 - Operadores Logicos/4.6.1 - Operador Logico de Conjuncao/andtest.kt
/*
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: 4 - Topico 4.6.1
 * 
 * Programa ..: OPERACAO DE CONJUNCAO ".E. / AND"
 * 
 */

import java.util.Scanner

fun main() {
  
    var entrada = Scanner(System.`in`)
    var NUMERO: Int

    println("TESTE LOGICO OPERADOR: AND\n")

    print("Entre um numero: ")
    NUMERO = entrada.nextInt()

    if (NUMERO >= 20 && NUMERO <= 90) {
        println("O numero esta na faixa de 20 a 90")
    } else {
        println("O numero esta fora da faixa de 20 a 90")
    }
	
}
